testing/trino-product-tests/src/main/resources/sql-tests/testcases/tpcds/q72.result

```diff
@@ -40,7 +40,7 @@ Active plants need necessary, widespread roads. Best back visits hold regularly
 Active plants need necessary, widespread roads. Best back visits hold regularly fresh friend|null|5209|0|1|1|
 Active windows shall not find small, relig|Conventional childr|5203|0|1|1|
 Actively different proceedings light yet so similar houses. Good circumstances shall not take only levels. Then moral pounds will clean very only national organisations. |null|5183|0|1|1|
-Activities say. Right lips resort current techniques. Regional, possible daughters might not present changes; students cannotice ridiculous, l|Conventional childr|5192|0|1|1|
+Activities say. Right lips resort current techniques. Regional, possible daughters might not present changes; students can notice ridiculous, l|Conventional childr|5192|0|1|1|
```

Comment:
HOw could this be broken?

Reply:
https://github.com/trinodb/trino/issues/4564